config/boards/nanopct4.conf

```diff
@@ -1,8 +1,11 @@
 # Rockchip RK3399 hexa core 4GB RAM SoC GBE USB3 USB-C WiFi/BT eMMC NVMe
 BOARD_NAME="NanoPC T4"
-BOARDFAMILY="rk3399"
+BOARDFAMILY="rockchip64"
```

Comment:
I think this change will only break legacy kernel. Is there any other reason for changing board family?